frontend/src/constants/index.ts

```diff
@@ -6,6 +6,7 @@ const ATOM_KEY = {
 
 const CALENDAR = {
   MAX_VIEW: 10,
+  EMPTY_TITLE: '제목없음',
```

Comment:
```suggestion
  EMPTY_TITLE: '(제목없음)',
```

<img width="234" alt="스크린샷 2022-08-18 13 48 24" src="https://user-images.githubusercontent.com/52729559/185296162-64e93e58-71c9-4037-8506-affa15787723.png">

다른 일정들과 구분할 수 있도록 괄호로 감싸는 건 어떨까요?

Reply:
`제목 없음` 으로 가겠습니다!